Bottom: 98d16480b7d93ea2391c05b32a592768b0e93cef
Top:    ad08e00a8f438f1a1a205c62d392402bf1470a18
Author: Ian Kent <ikent@redhat.com>
Date:   2013-10-17 10:22:01 +0800

Refresh of autofs-5.0.7-fix-automount-8-mailing-list-info.patch

---

diff --git a/man/automount.8 b/man/automount.8
index c1895c4..79e53f0 100644
--- a/man/automount.8
+++ b/man/automount.8
@@ -181,7 +181,7 @@ Don't know, I've fixed everything I know about.
 The documentation could be better.
 
 Please report other bugs along with a detailed description to
-<autofs@vger.kernel.org>. Please visit http://vger.kernel.org/vger-lists.html#autofs
+<autofs@vger.kernel.org>. Visit http://vger.kernel.org/vger-lists.html#autofs
 for information about the list.
 .SH AUTHOR
 H. Peter Anvin <hpa@transmeta.com> and Ian Kent <raven@themaw.net>.
